### DISCUSSION TITLE
Can I put Pinta's adjustments and effects into Gimp?

### DISCUSSION THREAD

**Discussion by AshleyPikachu** [PintaProject/Pinta#874] [Q&A]
I'm learning how to use gimp at the same time as Pinta and I thought it be great for work flow. I know gimp accepts plugins but I'm not sure where to begin.

**Comment by cameronwhite** [comment]
I suspect it would probably be (very) hard to do. I'm not super familiar with gimp's plugins, but with Pinta being written in C# it's probably quite difficult to bridge the two

**Reply by AshleyPikachu** [reply]
thanks for letting me know.
